src/utils/constants.ts

```diff
@@ -1,5 +1,7 @@
 // export const ARNS_NAME_REGEX = new RegExp('^(?!-)[a-z0-9-s+]{1,32}(?<!-)$');
-export const ARNS_NAME_REGEX = new RegExp('^[a-z0-9-s+]{1,32}$');
+export const ARNS_NAME_REGEX = new RegExp(
+  '^[a-zA-Z0-9][a-zA-Z0-9-]{0,30}[a-zA-Z0-9]$',
```

Comment:
change to '^([a-zA-Z0-9][a-zA-Z0-9-]{0,30}[a-zA-Z0-9]|[a-zA-Z0-9]{1})$' to allow for single char names